infra/image/system-service/fixnet.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Fix server IP in IPA Server
+Wants=network.target
+After=network.target
+Before=ipa.service
+
+[Service]
+Type=oneshot
+ExecStart=/root/fixnet.sh
+
```

Comment:
you could add:

StandardOutput=journal
StandardError=journal

if you want to capture the scripts errors in the journal logs.